zbus/src/address/percent.rs

```diff
@@ -0,0 +1,206 @@
+use std::{
+    borrow::Cow,
+    ffi::{OsStr, OsString},
+    fmt,
+};
+
+use super::{Error, Result};
+
+pub(crate) trait Encodable {
```

Comment:
Could you please add some docs to this type. Just a basic description is sufficient.

Reply:
// A trait for types that can be percent-encoded and written to a [`fmt::Formatter`].
 ok?

Reply:
yes